src/PythonMigrationViewExtension/GraphPythonDependencies.cs

```diff
@@ -28,23 +28,29 @@ internal GraphPythonDependencies(ViewLoadedParams viewLoadedParams)
             this.ViewLoaded = viewLoadedParams;
         }
 
-        internal bool ContainsIronPythonDependencies()
+        internal bool ContainsIronPythonDependency()
```

Comment:
hmm, I think a more appropriate name would be `CurrentWorkspaceContainsIronPythonDependency` - what do you think?

Reply:
yes, sounds good. will change.

Reply:
Sorry about being picky on this,, https://stackoverflow.com/questions/1370840/naming-conventions-what-to-name-a-method-that-returns-a-boolean

Maybe `ContainsIronPythonDependencyInCurrentWS`